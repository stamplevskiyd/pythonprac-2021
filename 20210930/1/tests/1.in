12,51,41,76,89,25,125
