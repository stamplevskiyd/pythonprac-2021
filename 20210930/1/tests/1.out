[51, 89, 25, 125, 12, 76, 41]
